### DISCUSSION TITLE
Composite Key on a Table

### DISCUSSION THREAD

**Discussion by emeraldjava** [supabase/supabase#1345] [Questions] [answered]
Hi,
Is it possible to define multiple columns as a composite key using the basic supabase offering. 
Thanks,
emeraldjava

**Comment by inian** [comment, marked as answer]
Yes. Anything thats possible with postgres, you can do with Supabase too. Just run the query in the sql editor. Are you looking for help in creating the composite key?

**Reply by emeraldjava** [reply]
No - i guess i was wondering was there some magic feature via the UI to set composite keys. I'm happy to do it via SQL and the command line.
